src/nodeconductor_test/helpers.py

```diff
@@ -62,6 +62,14 @@ def _back_to_list(driver):
     time.sleep(BaseSettings.click_time_wait)
 
 
+def _create_provider(driver):
```

Comment:
Function name is confusing. It is not clear from name that function will only click on button. Suggested name: `_go_to_provider_create_page`